ts-client/tests/ValorClient.test.ts

```diff
@@ -227,14 +246,14 @@ test('evaluation methods', async () => {
   );
   // check we can get evaluations by model names
   expect((await client.getEvaluationsByModelNames([modelNames[0]])).length).toBe(2);
-  expect((await client.getEvaluationsByModelNames(modelNames)).length).toBe(4);
+  expect(
+    (await client.getEvaluationsByModelNames(modelNames, ['mAccuracy'])).length
```

Comment:
added a quick test to make sure metricsToSortBy runs as expected. didn't add detailed tests here as a) all of these evaluations have the same mAccuracy and b) I'd already written detailed integration tests in the python client